.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -13,14 +13,6 @@ parameters:
     displayName: Branch in aks-rp to use for overrides
     type: string
     default: master
-  - name: build1804containerd
-    displayName: Build 1804 containerd
-    type: boolean
-    default: true
-  - name: build1804gen2containerd
```

Comment:
you also need to freeze the SIG image config templates in sig_config.go to the currently-latest version: https://github.com/Azure/AgentBaker/blob/master/pkg/agent/datamodel/sig_config.go#L479